app/app.go

```diff
@@ -357,14 +365,26 @@ func New(
 		),
 	)
 
+	// IBC Fee Module keeper
+	app.IBCFeeKeeper = ibcfeekeeper.NewKeeper(
+		appCodec, keys[ibcfeetypes.StoreKey], app.GetSubspace(ibcfeetypes.ModuleName),
+		app.IBCFeeKeeper, // may be replaced with IBC middleware
```

Comment:
so IBC middleware isn't there yet?

Reply:
We can add more middlewares in the future. I'll modify the comment.